### PR TITLE
added header

### PR DESCRIPTION
**Pull request by herzlicher-scheiss**

### CODE REVIEW

**Comment by herzlicher-scheiss** [comment]
rewrited

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+    <title>Hello world!</title>
+    <link rel="stylesheet" href="style.css" >
+  </head>
+  <body>
+    <h1>Hello world!</h1>
```
Comment (Virass) [inline src/index.html:11]: h1 tag isn't need here

**Review by Virass — CHANGES_REQUESTED** [review]
Fix you homework and add demo links here

image

**Comment by herzlicher-scheiss** [comment]
https://herzlicher-scheiss.github.io/layout_hello-world/
https://herzlicher-scheiss.github.io/layout_hello-world/report/html_report/

**Pull request closed without merging**